code/modules/surgery/organs/heart.dm

```diff
@@ -107,13 +107,14 @@
 	actions_types = list(/datum/action/item_action/organ_action/cursed_heart)
 	var/last_pump = 0
 	var/add_colour = TRUE //So we're not constantly recreating colour datums
-	var/pump_delay = 30 //you can pump 1 second early, for lag, but no more (otherwise you could spam heal)
-	var/blood_loss = 100 //600 blood is human default, so 5 failures (below 122 blood is where humans die because reasons?)
+	var/pump_delay = 70 //you can pump 1 second early, for lag, but no more (otherwise you could spam heal)
+	var/blood_regained = 70
 
 	//How much to heal per pump, negative numbers would HURT the player
-	var/heal_brute = 0
-	var/heal_burn = 0
-	var/heal_oxy = 0
+	var/heal_brute = 25
+	var/heal_burn = 25
+	var/heal_oxy = 10
+	var/heal_toxin = 10
```

Comment:
lets not make miners heal toxin freely

Reply:
https://github.com/tgstation/tgstation/pull/68739/commits/e6fb9393dfc3cb3f2e5c0a1e43fbec85ba2f6066